packages-web/datagrid/src/components/Table.tsx

```diff
@@ -199,14 +205,23 @@ export function Table<T>(props: TableProps<T>): ReactElement {
                                 <Header
                                     column={column}
                                     key={`headers_column_${index}`}
-                                    sortable={props.columnsSortable}
-                                    resizable={props.columnsResizable}
-                                    filterable={props.columnsFilterable}
                                     draggable={props.columnsDraggable}
-                                    visibleColumns={visibleColumns}
+                                    dragOver={dragOver}
+                                    filterable={props.columnsFilterable}
+                                    resizable={props.columnsResizable}
                                     setColumnOrder={setColumnOrder}
+                                    setDragOver={setDragOver}
+                                    sortable={props.columnsSortable}
+                                    visibleColumns={visibleColumns}
                                 />
                             ))}
+                            {props.columnsHidable && (
```

Comment:
You changed the place of ColumnSelector, was this intentional :) ?

Reply:
Yes, now its inside the column and not in the header anymore.

Reply:
<img width="604" alt="image" src="https://user-images.githubusercontent.com/45102481/86918111-7e4b3f00-c126-11ea-8f68-b9b6919379db.png">